sass/components/mixins/brand.scss

```diff
@@ -0,0 +1,11 @@
+@mixin create-brand($brand-font-size: $default-brand-font-size, $brand-font-color: $brand-font-color){
```

Comment:
we have 2 naming convention in this: $default-brand-font-size and $brand-font-color